GUI/src/cz/fidentis/gui/actions/analysis/AnalysisActionDialogue.java

```diff
@@ -0,0 +1,336 @@
+        DefaultListModel model = (DefaultListModel) jListCD.getModel();
+        
+        if(jListCD.getSelectedIndex() != -1){
+            selectedFilesSecond.remove(jListCD.getSelectedIndex());
+            model.remove(jListCD.getSelectedIndex());
+        }
+    }//GEN-LAST:event_removeComprButtonActionPerformed
+
+    private void computeButtonActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_computeButtonActionPerformed
+
+        //make new window for computation
+        final AnalysisResults dialog = new AnalysisResults(selectedFiles, selectedFilesSecond, computeComboBox.getSelectedIndex());
+        JFrame frame = new JFrame();
+        frame.add(dialog);
+        frame.setSize(dialog.getPreferredSize());
+        frame.setVisible(true); 
+    }//GEN-LAST:event_computeButtonActionPerformed
+
+    private void computeComboBoxActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_computeComboBoxActionPerformed
+        // TODO add your handling code here:
+    }//GEN-LAST:event_computeComboBoxActionPerformed
+
+    // Variables declaration - do not modify//GEN-BEGIN:variables
+    private javax.swing.JButton addComparisonDataButton;
+    private javax.swing.JButton addGroundThruthButton;
+    private javax.swing.ButtonGroup buttonGroup1;
+    private javax.swing.JButton computeButton;
+    private javax.swing.JComboBox<String> computeComboBox;
+    private javax.swing.JList<String> jListCD;
+    private javax.swing.JList<String> jListGT;
+    private javax.swing.JPanel jPanel1;
+    private javax.swing.JScrollPane jScrollPane1;
+    private javax.swing.JScrollPane jScrollPane2;
+    private java.awt.Label label1;
+    private java.awt.Label label2;
+    private java.awt.Label label3;
+    private java.awt.Label label4;
+    private javax.swing.JButton removeComprButton;
+    private javax.swing.JButton removeGroundTButton;
+    // End of variables declaration//GEN-END:variables
+    @Override
+    public void componentOpened() {
+        // TODO add custom code on component opening
+    }
+
+    @Override
+    public void componentClosed() {
+        // TODO add custom code on component closing
+    }
+
+    void writeProperties(java.util.Properties p) {
+        // better to version settings since initial version as advocated at
+        // http://wiki.apidesign.org/wiki/PropertyFiles
+        p.setProperty("version", "1.0");
+        // TODO store your settings
+    }
+
+    void readProperties(java.util.Properties p) {
+        String version = p.getProperty("version");
+        // TODO read your settings according to their version
+    }
+    
+    /**
+     * Handle adding files for comparison
+     * 
+     * @param fpPoints list of selected files
+     * @param selected list of all selected files
+     * @param jList jList for window selection
+     */
+    private void addFilesHandler(List<FpModel> fpPoints, List<FpModel> selected, javax.swing.JList<String> jList){
```

Comment:
je toto rovnaka metoda ako v triede LandmarkTrainDialogueTopComponent.java? ak ano, dalo by sa to refaktorovat do vlastnej triedy, nech to tu nie je 2x?